core/src/main/scala/org/typelevel/sbt/TypelevelPlugin.scala

```diff
@@ -110,10 +110,4 @@ object TypelevelPlugin extends AutoPlugin {
 
   // override for bincompat
   override def projectSettings = immutable.Seq.empty
-
-  private val primaryJavaCond = Def.setting {
-    val java = githubWorkflowJavaVersions.value.head
-    s"matrix.java == '${java.render}'"
-  }
-
```

Comment:
Unused. Seems it was copied to here:
https://github.com/typelevel/sbt-typelevel/blob/c810fa9f8d3d71c9fc18158537699ac7f62891e1/ci/src/main/scala/org/typelevel/sbt/TypelevelCiPlugin.scala#L129-L132

Reply:
Nice!